examples/user_guide/12-Responding_to_Events.ipynb

```diff
@@ -372,15 +425,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "style.color_levels = 10\n",
-    "style.cmap = 'plasma'"
+    "overdraft_limits = {'student':300, 'regular':100, 'savings':0} # Overdraft limits for different account types\n",
+    "streams = dict(owner=account.param.name, total=slider.param.value, acc=checkbox.param.value)\n",
+    "\n",
+    "def account_info(owner, total, acc):\n",
+    "    return hv.Table([(owner, acc, overdraft_limits[acc], total)], \n",
+    "                    ['Owner', 'Account Type', 'Overdraft ($)', 'Total ($)'])\n",
+    "\n",
+    "widget_dmap = hv.DynamicMap(account_info, streams=streams)\n",
+    "widget_dmap.opts(height=100)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "It is also possible to be even more explicit and directly map parameters to `.apply` keywords without manually constructing a stream, e.g. the example may also be written as:"
+    "\n",
+    "You can now update the plot above using the slider and dropdown widgets. Note that for all these examples, a `Params` stream is created internally. This type of stream can wrap parameterized objects or sets of parameters but it is rare that it needs to be used directly at the user level (though this was necessary in HoloViews < 1.10.8).  To see more examples of how to use `panel` with HoloViews, see the [Dashboards user guide](./16-Dashboards.ipynb).\n",
```

Comment:
```suggestion
    "You can now update the plot above using the slider and dropdown widgets. Note that for all these examples, a `Params` stream is created internally. This type of stream can wrap Parameterized objects or sets of Parameters but (since HoloViews 1.10.8) it is rare that an explicit stream object like that needs to be used directly at the user level.  To see more examples of how to use Panel with HoloViews, see the [Dashboards user guide](./16-Dashboards.ipynb).\n",
```